src/components/List/StatusCell.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Grid, Icon, Label, Progress } from 'semantic-ui-react'
+import { ApplicationStatus } from '../../utils/generated/graphql'
+import { ApplicationStage, CellProps } from '../../utils/types'
+
+enum ACTIONS {
+  EDIT_DRAFT = 'Edit draft',
+  MAKE_CHANGES = 'Make changes',
+  RENEW = 'Renew',
+}
+
+const StatusCell: React.FC<CellProps> = ({ application }) => {
+  const { serial, stage } = application
+  const status = stage?.status
+  switch (status) {
```

Comment:
I don't seem to be getting the correct `status` here. Made a new CompRego application, status is showing as draft in database. The application shows in the list but the Status field is blank.
Console output for serial, stage, status:
```
Status 8259 
Object { id: 2, name: "Screening", status: undefined, date: "2021-01-19" } undefined
```

Reply:
I will investigate this (and try to remove the Context) will get back to you when finished.

Reply:
Any joy with this? (I've noticed a few things when doing the filtering, if you wanted to discuss it first?)

Reply:
Have solved this problem now. And removed the use of `ListContext`